app/controllers/formbuilder/edit.js

```diff
@@ -18,10 +18,9 @@ export const GRAPHS = {
 const SOURCE_NODE = new RDFNode('http://frontend.poc.form.builder/sourcenode');
 
 export default class FormbuilderEditController extends Controller {
```

Comment:
Some methods / tasks / actions can probably be grouped better or reordered. I skipped the cleanup part of the refactor because I ran out of time. 😅

Reply:
I think it's already very clean ! But I don't know what magic you where thinking of :blush: